### DISCUSSION TITLE
Question: can I adjust the whitespace/gap between preview window and fzf?

### DISCUSSION THREAD

**Discussion by linrongbin16** [junegunn/fzf#3615] [Q&A] [answered]
<img width="1280" alt="image" src="https://github.com/junegunn/fzf/assets/6496887/04c927d7-dde1-4dec-95d3-125a724ee90c">

As we can see in screen shot, there's some whitespace/gap between preview window and fzf.

I already know that there's `--padding` and `--margin` option to adjust the gaps, but I think they are for the whole fzf, not for preview window?

**Comment by junegunn** [comment, marked as answer]
No, there's no such option. The screenshot _appears_ to have some extra margin, not because of an option, but because the entries shown are short for the width.
